tasks/src/tabular-regression/about.md

```diff
@@ -36,8 +28,48 @@ model.fit(X, y)
 ```
 
 
+# Model Hosting and Inference
+
+You can use [skops](https://skops.readthedocs.io/) for model hosting and inference on 🤗Hub. Using `skops` you can:
+- Easily use inference API,
+- Build Gradio UIs with one line of code,
+- Programmatically create model cards,
+- Securely serialize your scikit-learn model. (See limitations of using pickle [here](https://huggingface.co/docs/hub/security-pickle).)
```

Comment:
Same comments as for tabular classification

Reply:
the benchmark already has those datasets.